docs/user/orion.rst

```diff
@@ -0,0 +1,97 @@
+^^^^^^^^
+
+Orion's settings are at *Settings -> Tools -> Orion*. The settings include:
+
+- The Artemis base url. Can be changed to switch to a specific Artemis instance. Defaults to https://artemis.ase.in.tum.de. **Important:** The url must not end with a /, otherwise it does not work!
+- The Artemis exercise paths. Orion suggests to store newly downloaded exercises at ``default-path/course/exercise-name``, with the default path dependant of the setting.
+- The default commit message
+- An option to change the user agent if needed. The user agent is sent to Artemis to identify Orion. I needs to contain "Orion/*orion-version*". Usually, no changes are required.
+- A button to restart the integrated browser. If Orion gets stuck, restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_.
+
+Participation in Orion
+----------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, click |open-in-intellij-button|.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. To submit the changes, click |submit-button|. This automatically commits, pushes the local changes, and displays the result of the automatic tests in IntelliJ.
+
+Gif showcasing the usage for students:
+
+  .. image:: https://raw.githubusercontent.com/ls1intum/Orion/master/.github/media/orion_workflow.gif
+            :align: center
+            :width: 100%
+
+Exercise Creation in Orion
+--------------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the exercise creation of programming exercises, step 1 and 2.
+ 2. After the creation, navigate to the instructor exercise overview using the integrated browser
+ 3. Each programming exercise provides a button to edit the exercise in Orion |edit-in-intellij-button|. The button is rightmost in the table an might require scrolling. This downloads the template, solution and test repository of the exercise.
+ 4. Edit the repository files in IntelliJ.
+ 5. To submit the changes, click |submit-button|. This commits and pushes all local changes to their respective repository.
+ 6. The integrated browser displays the editor to edit the problem statement.
+ 7. To test the code locally, click |test-locally-button|, which copies the tests with the local template or solution (whichever was selected) into a new folder and execute them locally.
+
+Assessment in Orion
+-------------------
+
+Tutors can assess programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the assessment dashboard of the exercise using the integrated browser.
+ 2. Click |assess-in-orion-button|. This automatically downloads the automatic tests.
```

Comment:
how can the tests be downloaded? it downloads a repository/folder containing all the specified tests right? That could be described as such...

Reply:
I added "repository with the tests". I was a bit hesistant since technically it also downloads the template, but the user does not need to know that. Perhaps telling it sets up the project is sufficient for the user, what do you think?

Reply:
yes, "sets up project" sounds good